src/commons/application/ApplicationTypes.ts

```diff
@@ -273,9 +287,16 @@ export const defaultAchievement: AchievementState = {
   assessmentOverviews: []
 };
 
+export const defaultLanguageConfig: SALanguage = ALL_LANGUAGES.find(
+  sublang =>
+    sublang.chapter === Constants.defaultSourceChapter &&
+    sublang.variant === Constants.defaultSourceVariant
+)!;
```

Comment:
I really don't like the non-null assertion operator. It might be the case that the default Source chapter & variant will always have a valid `SALanguage` now, but we cannot guarantee this to be true in the future. A preferable alternative would be to add a guard clause that throws an error so that if and when this invariant is broken in the future, we know exactly what the problem is.

Reply:
Noted and agreed, will make the changes. Thanks!

Reply:
Resolved in 2013b7d1bee0909e5a6c7b9d7a64ebbabb4c9dec.